### ISSUE TITLE
Dashboard Login did not work

### ISSUE THREAD

**Issue by dcaud** [fruxefarms/FruxePi#4]
I believe I followed the complete installation instructions correctly and saw no errors throughout the process. However, when I opened a web browser on the PI and went to http://my-raspi-ip-address:8080/  I received the following error:

my-raspi-ip-address refused to connect

I've done a quick search for how to solve this, but I've found nothing yet.

Any ideas?

[I entered the IP address given by `ip add` for my-raspi-ip-address...something like 192.168.2.19]

**Comment by dcaud** [comment]
I think the problem was that the instructions for the Apache access were made for a prior version of Apache.

On my machine, Apache 2.4 is running.

Because of a difference between 2.2 and 2.4, I think the following:

```
<Directory /var/www/>
    Options Indexes FollowSymLinks MultiViews
    AllowOverride All
    Order allow,deny
    allow from all
</Directory>
```

Should be changed to this:

```
<Directory /var/www/>
    Options Indexes FollowSymLinks MultiViews
    AllowOverride All
    Require all granted
</Directory>
```

**Comment by jkissner** [comment]
You did find the correct fix for the config in Apache 2.2-2.4.  Also, the instructions denote using port 8080, this should actually just be 80 (no port entered).

**Comment by fruxefarms** [comment]
Adjusted the documentation to reflect @dcaud suggestion as well as removed port 8080 from documentation per @jkissner request. Thanks for the help!
